qdb/scripts/formatter.py

```diff
@@ -266,7 +345,7 @@ def generate_report(data, filename, today=None):
         adjust_row_heights(ws, row)
         Worksheet.set_printer_settings(ws, paper_size=1, orientation="landscape")
     if len(data["sub02s"]) > 0:
-        row = add_sub02_tab(wb, data, tab_index + 1)
+        row = add_sub02_tab(wb, data, len(data["accounts"]))
```

Comment:
This change of the 3rd parameter from `tab_index + 1` to `len(data["accounts"])` confused me, though I see it is correct and resolves a "possibly unbound" linter message.  I added a clarifying comment and explicit parameter:
```
        # The sub02 data needs to go in a sheet (tab) after the accounts data.
        row = add_sub02_tab(wb, data, tab_index=len(data["accounts"]))
```